XUnitTestProject/Logic/MappingTestsFixture.cs

```diff
@@ -0,0 +1,23 @@
+﻿using AutoMapper;
+using IssueTracker.ApiConfig;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+

```

Comment:
remove space here. otherwise great job with all the changes and naming. also extension methods makes it so much cleaner